packages/web/src/index.css

```
@@ -1,134 +1,287 @@
+    min-height: 100%;
+  }
 
-body {
-  font-family: system-ui, -apple-system, sans-serif;
-  background-color: hsl(var(--background));
-  color: hsl(var(--foreground));
-  margin: 0;
-  min-height: 100vh;
+  body {
+    @apply bg-background text-foreground antialiased;
+    margin: 0;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)